src/pages/privacy-policy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Privacy Policy'
+layout: '~/layouts/MarkdownLayout.astro'
+---
+
+### Privacy Policy
+
+LycheeOrg is committed to protecting the privacy of its users. This Privacy Policy outlines how we handle personal data and what measures we take to ensure your privacy.
+
+#### On LycheeOrg and pr.lycheeorg.dev
+
+- LycheeOrg does not collect any personal data from its users.
+- LycheeOrg does not use cookies or any other tracking technologies.
+- LycheeOrg does not share any data with third parties.
+- LycheeOrg does not store any data on its servers.
+
+#### On weblate.lycheeorg.dev
+
+Weblate is used to translate Lychee. It is a third-party service that is self-hosted by us.
+To the best of our knowledge, Weblate does not collect any personal data from its users aside from their mail address
```

Comment:
```suggestion
To the best of our knowledge, the Weblate software does not collect any personal data from its users aside from their email address
```

Just trying to make it clear we're talking about the product not the organisation. Unless you meant the org here?